js/src/setup-ads/ads-stepper/index.js

```diff
@@ -57,6 +57,10 @@ const AdsStepper = ( { formProps } ) => {
 		continueStep( '2' );
 	};
 
+	// @todo: Add check for billing status once billing setup is moved to step 2.
```

Comment:
Checking billing status cannot be done here, it must instead be done after an Ads account is connected in Step 1. The billing status check will be handled in `AdsCampaign` after #2535 is completed, so we can ignore this check for now.